packages/icons/web/v2/pix_full_24.svg

```diff
@@ -0,0 +1,23 @@
+<svg version="1.1" id="pix_full_24_svg__Layer_1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 600 600" style="enable-background:new 0 0 600 600" xml:space="preserve">
```

Comment:
The SVG should be 24px tall and maintain the original aspect ratio of the logo. There should be no whitespace around the graphic.